<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="MSI" failures="14" tests="42" name="Test" time="412.779" errors="0" timestamp="2023-08-03T15:21:52 IST">
  <testcase classname="tests.LoginTest" name="checkLogin" time="0.58"/>
  <testcase classname="tests.LoginTest" name="checkLogin" time="0.48"/>
  <testcase classname="tests.LoginTest" name="checkLogin" time="5.486"/>
  <testcase classname="tests.LoginTest" name="checkLogin" time="0.365"/>
  <testcase classname="tests.HomeTest" name="checkLogOut" time="1.2"/>
  <testcase classname="tests.AddToCartTest" name="addToCart" time="12.983"/>
  <testcase classname="tests.AddToCartTest" name="sortProducts" time="0.639"/>
  <testcase classname="tests.AddToCartTest" name="sortProducts" time="0.89">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.AddToCartTest.sortProducts(AddToCartTest.java:71)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- sortProducts -->
  <testcase classname="tests.AddToCartTest" name="sortProducts" time="10.65"/>
  <testcase classname="tests.AddToCartTest" name="sortProducts" time="0.573"/>
  <testcase classname="tests.AddToCartTest" name="click_Highest_Product" time="5.76"/>
  <testcase classname="tests.AddToCartTest" name="click_Highest_Product" time="16.085">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.AddToCartTest.click_Highest_Product(AddToCartTest.java:111)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- click_Highest_Product -->
  <testcase classname="tests.AddToCartTest" name="click_Highest_Product" time="15.816"/>
  <testcase classname="tests.AddToCartTest" name="click_Highest_Product" time="5.389">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.AddToCartTest.click_Highest_Product(AddToCartTest.java:111)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- click_Highest_Product -->
  <testcase classname="tests.AddToCartTest" name="verifyProductDetails" time="5.564"/>
  <testcase classname="tests.AddToCartTest" name="verifyProductDetails" time="5.461"/>
  <testcase classname="tests.AddToCartTest" name="verifyProductDetails" time="10.435"/>
  <testcase classname="tests.AddToCartTest" name="verifyProductDetails" time="5.378">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.AddToCartTest.verifyProductDetails(AddToCartTest.java:133)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verifyProductDetails -->
  <testcase classname="tests.AddToCartTest" name="checkOut_Products_Validation" time="0.839"/>
  <testcase classname="tests.AddToCartTest" name="checkOut_Products_Validation" time="0.903">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.AddToCartTest.checkOut_Products_Validation(AddToCartTest.java:163)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- checkOut_Products_Validation -->
  <testcase classname="tests.AddToCartTest" name="checkOut_Products_Validation" time="5.853"/>
  <testcase classname="tests.AddToCartTest" name="checkOut_Products_Validation" time="10.795"/>
  <testcase classname="tests.CheckOutTest" name="checkOut" time="4.312"/>
  <testcase classname="tests.CheckOutTest" name="checkOut" time="4.287">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.CheckOutTest.checkOut(CheckOutTest.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- checkOut -->
  <testcase classname="tests.CheckOutTest" name="checkOut" time="9.26"/>
  <testcase classname="tests.CheckOutTest" name="checkOut" time="10.766">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.CheckOutTest.checkOut(CheckOutTest.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- checkOut -->
  <testcase classname="tests.CheckOutTest" name="checkTotal_Prod_WithCartItems" time="6.397"/>
  <testcase classname="tests.CheckOutTest" name="checkTotal_Prod_WithCartItems" time="6.226">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.CheckOutTest.checkTotal_Prod_WithCartItems(CheckOutTest.java:62)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- checkTotal_Prod_WithCartItems -->
  <testcase classname="tests.CheckOutTest" name="checkTotal_Prod_WithCartItems" time="11.41"/>
  <testcase classname="tests.CheckOutTest" name="checkTotal_Prod_WithCartItems" time="12.745">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.CheckOutTest.checkTotal_Prod_WithCartItems(CheckOutTest.java:62)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- checkTotal_Prod_WithCartItems -->
  <testcase classname="tests.CheckOutTest" name="checkFieldInformation" time="3.085"/>
  <testcase classname="tests.CheckOutTest" name="checkFieldInformation" time="2.989"/>
  <testcase classname="tests.CheckOutTest" name="checkFieldInformation" time="8.135"/>
  <testcase classname="tests.CheckOutTest" name="checkFieldInformation" time="10.768">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.CheckOutTest.checkFieldInformation(CheckOutTest.java:99)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- checkFieldInformation -->
  <testcase classname="tests.CheckOutTest" name="checkOrderCompletion" time="8.575"/>
  <testcase classname="tests.CheckOutTest" name="checkOrderCompletion" time="8.365">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.CheckOutTest.checkOrderCompletion(CheckOutTest.java:140)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- checkOrderCompletion -->
  <testcase classname="tests.CheckOutTest" name="checkOrderCompletion" time="13.55"/>
  <testcase classname="tests.CheckOutTest" name="checkOrderCompletion" time="12.764">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.CheckOutTest.checkOrderCompletion(CheckOutTest.java:140)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- checkOrderCompletion -->
  <testcase classname="tests.CheckOutTest" name="checkBackToHome" time="6.45"/>
  <testcase classname="tests.CheckOutTest" name="checkBackToHome" time="4.22">
    <failure type="java.lang.AssertionError" message=" User handling the exception, couldnt reach the checkout page expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError:  User handling the exception, couldnt reach the checkout page expected [true] but found [false]
at tests.CheckOutTest.checkBackToHome(CheckOutTest.java:184)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- checkBackToHome -->
  <testcase classname="tests.CheckOutTest" name="checkBackToHome" time="16.48"/>
  <testcase classname="tests.CheckOutTest" name="checkBackToHome" time="10.702">
    <failure type="java.lang.AssertionError" message=" User handling the exception, couldnt reach the checkout page expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError:  User handling the exception, couldnt reach the checkout page expected [true] but found [false]
at tests.CheckOutTest.checkBackToHome(CheckOutTest.java:184)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- checkBackToHome -->
  <testcase classname="driverFactory.BaseClass" name="@BeforeTest init" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
  <testcase classname="driverFactory.BaseClass" name="@BeforeTest init" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
  <testcase classname="driverFactory.BaseClass" name="@BeforeTest init" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
</testsuite> <!-- Test -->
